_data/committers.yml

```diff
@@ -218,7 +218,7 @@
 - name: Wes McKinney
   role: PMC
   alias: wesm
-  affiliation: Voltron Data
+  affiliation: Posit
```

Comment:
```suggestion
  affiliation: Posit PBC
```

Reply:
We wrote "Posit" for other committers. Is "PBC" a meaningful addition or is it just like "Inc" or "Corp"?

Reply:
The latter. Was basing my suggestion off the comment in https://github.com/apache/arrow-site/pull/548#discussion_r1801264169. Your call though.

Reply:
Just "Posit" is fine